### PR TITLE
support SQL contains non-ascii characters

### PR DESCRIPTION
**Pull request by osiire**
this is a PR to improve issue #967 

- simple solution using UNICODE_CHARACTER_CLASS.
- no impact to current behavior unless parsing huge SQL sentence. 

thank you.

### CODE REVIEW

**Review by seratch — APPROVED** [review]
Looks great to me! Thanks for improving this 👍

**Comment by seratch** [comment]
We'll merge this PR once the CI builds pass

File: scalikejdbc-core/src/test/scala/scalikejdbc/SQLTemplateParserSpec.scala [hunk 1]
```diff
@@ -249,4 +249,11 @@ users"""
     SQLTemplateParser.convertToSQLWithPlaceHolders(sql) should equal(expected)
   }
 
+  //https://github.com/scalikejdbc/scalikejdbc/issues/967
+  it should "support SQL contains non-ascii characters" in {
+    val sql = "SELECT 名前 FROM ユーザ WHERE user_id = {id})"
+    val params = SQLTemplateParser.extractAllParameters(sql)
+    params.size should equal(1)
+    params.head should equal('id)
```
Comment (xuwei-k) [inline scalikejdbc-core/src/test/scala/scalikejdbc/SQLTemplateParserSpec.scala:257]:
```suggestion
    params.head should equal("id")
```

Comment (seratch) [reply scalikejdbc-core/src/test/scala/scalikejdbc/SQLTemplateParserSpec.scala]: Good catch - I didn't know ScalaTest assertion passes with both

**Pull request merged**